### PR TITLE
New working clones

### PR DESCRIPTION
**Pull request by clawgrip**
------------------
Red Hawk (horizontal, Spain) [Victor Fernandez (City Game), ClawGrip]

* Reformated comments as single-line style
* Fixed 'redhawki' game name (is 'Hae Dong', not 'Hea Dong')

### CODE REVIEW

File: src/mame/drivers/nmk16.cpp [hunk 1]
```diff
@@ -5702,124 +5702,118 @@ static void decryptcode( running_machine &machine, int a23, int a22, int a21, in
...
-
+	ROM_LOAD16_BYTE( "vdk-07.202",  0x000000, 0x080000, CRC(42d41f06) SHA1(69fd1d38187b8081f65acea2424bc1a0d455d90c) )  // Sprites
+	ROM_LOAD16_BYTE( "vdk-06.203",  0x000001, 0x080000, CRC(d54722a8) SHA1(47f8e97b29ae0ff1a1d7d50734e4219a87a2ed57) )  // Sprites
+	ROM_LOAD16_BYTE( "vdk-04.2-1",  0x100000, 0x080000, CRC(0a730547) SHA1(afac0549eb86d1fab5ca8ae2a0dad14144f55c02) )  // Sprites
+	ROM_LOAD16_BYTE( "vdk-05.3-1",  0x100001, 0x080000, CRC(ba456d27) SHA1(5485a560ae2c2c8b6fdec314393c02a3de758ef3) )  // Sprites
+
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "vdk-02.126",     0x000000, 0x080000, CRC(b2103274) SHA1(6bbdc912393607cd5306be946327c5ea0178c7a6) ) // 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "vdk-03.165",     0x000000, 0x080000, CRC(631776d3) SHA1(ffd76e5b03130252c55eaa6ae7edfee5632dae73) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:5785]: Why have you removed blank lines visually separating the ROM regions?

File: src/mame/drivers/nmk16.cpp [hunk 2]
```diff
@@ -5702,124 +5702,118 @@ static void decryptcode( running_machine &machine, int a23, int a22, int a21, in
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:5727]: The blank lines here separated the ROM regions, too.

File: src/mame/drivers/nmk16.cpp [hunk 3]
```diff
@@ -5702,124 +5702,118 @@ static void decryptcode( running_machine &machine, int a23, int a22, int a21, in
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:5756]: Also here.

File: src/mame/drivers/nmk16.cpp [hunk 4]
```diff
@@ -5910,23 +5904,21 @@ ROM_START( mustang )
...
-	ROM_REGION( 0x080000, "oki1", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "90058-5",    0x00000, 0x80000, CRC(c60c883e) SHA1(8a01950cad820b2e781ec81cd12737829edc4f19) ) /* 0x20000 - 0x80000 banked */
-
-	ROM_REGION( 0x080000, "oki2", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "90058-6",    0x00000, 0x80000, CRC(233c1776) SHA1(7010a2f914611698a65bf4f22bc1753a9ed26277) ) /* 0x20000 - 0x80000 banked */
-
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "90058-5",    0x00000, 0x80000, CRC(c60c883e) SHA1(8a01950cad820b2e781ec81cd12737829edc4f19) ) // 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "90058-6",    0x00000, 0x80000, CRC(233c1776) SHA1(7010a2f914611698a65bf4f22bc1753a9ed26277) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:5910]: And here.

File: src/mame/drivers/nmk16.cpp [hunk 5]
```diff
@@ -5939,23 +5931,21 @@ ROM_START( mustangs )
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:5937]: Here as well.

File: src/mame/drivers/nmk16.cpp [hunk 6]
```diff
@@ -6013,27 +6003,25 @@ ROM_START( acrobatm )
...
-	ROM_REGION( 0x80000, "oki1", 0 )    /* OKIM6295 samples */
-	ROM_LOAD( "05_ic54.bin",    0x00000, 0x80000, CRC(3b8c2b0e) SHA1(72491da32512823540b67dc5027f21c74af08c7d) ) /* 0x20000 - 0x80000 banked */
-
-	ROM_REGION( 0x80000, "oki2", 0 )    /* OKIM6295 samples */
-	ROM_LOAD( "06_ic53.bin",    0x00000, 0x80000, CRC(c1517cd4) SHA1(5a91ddc608c7a6fbdd9f93e503d39eac02ef04a4) ) /* 0x20000 - 0x80000 banked */
-
+	ROM_REGION( 0x80000, "oki1", 0 )    // OKIM6295 samples
+	ROM_LOAD( "05_ic54.bin",    0x00000, 0x80000, CRC(3b8c2b0e) SHA1(72491da32512823540b67dc5027f21c74af08c7d) ) // 0x20000 - 0x80000 banked
+	ROM_REGION( 0x80000, "oki2", 0 )    // OKIM6295 samples
+	ROM_LOAD( "06_ic53.bin",    0x00000, 0x80000, CRC(c1517cd4) SHA1(5a91ddc608c7a6fbdd9f93e503d39eac02ef04a4) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6021]: And here.

File: src/mame/drivers/nmk16.cpp [hunk 7]
```diff
@@ -6089,34 +6077,32 @@ ROM_START( bioship )
...
-	ROM_REGION(0x80000, "oki1", 0 ) /* Oki sample data */
-	ROM_LOAD( "sbs-g_04.ic139",    0x00000, 0x80000, CRC(7c74cc4e) SHA1(92097b372eacabdb9e8e261b0bc4223821ff9273) ) /* 0x20000 - 0x80000 banked */
-
-	ROM_REGION(0x80000, "oki2", 0 ) /* Oki sample data */
-	ROM_LOAD( "sbs-g_05.ic160",    0x00000, 0x80000, CRC(f0a782e3) SHA1(d572226b8e597f1c34d246cb284e047a6e2d9290) ) /* 0x20000 - 0x80000 banked */
-
+	ROM_REGION(0x80000, "oki1", 0 ) // Oki sample data
+	ROM_LOAD( "sbs-g_04.ic139",    0x00000, 0x80000, CRC(7c74cc4e) SHA1(92097b372eacabdb9e8e261b0bc4223821ff9273) ) // 0x20000 - 0x80000 banked
+	ROM_REGION(0x80000, "oki2", 0 ) // Oki sample data
+	ROM_LOAD( "sbs-g_05.ic160",    0x00000, 0x80000, CRC(f0a782e3) SHA1(d572226b8e597f1c34d246cb284e047a6e2d9290) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6101]: Here as well.

File: src/mame/drivers/nmk16.cpp [hunk 8]
```diff
@@ -6125,170 +6111,160 @@ ROM_START( sbsgomo )
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6135]: And here.

File: src/mame/drivers/nmk16.cpp [hunk 9]
```diff
@@ -6125,170 +6111,160 @@ ROM_START( sbsgomo )
...
-	ROM_LOAD( "90068-2.bin", 0x00000, 0x80000, CRC(3a583184) SHA1(9226f1ea7725e4b48bb055d1c17389cf960d75f8) ) /* 0x20000 - 0x80000 banked */
-
-	ROM_REGION( 0x080000, "oki2", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "90068-1.bin", 0x00000, 0x80000, CRC(e7af69d2) SHA1(da050880e186954bcf0e0adf00750dd5a371551b) ) /* 0x20000 - 0x80000 banked */
+	ROM_LOAD16_WORD_SWAP( "90068-8.bin", 0x000000, 0x100000, CRC(9d3204b2) SHA1(b37a246ad37f9ce092b371f01122ddf2bc8b2db6) ) // Sprites
 
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "90068-2.bin", 0x00000, 0x80000, CRC(3a583184) SHA1(9226f1ea7725e4b48bb055d1c17389cf960d75f8) ) // 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "90068-1.bin", 0x00000, 0x80000, CRC(e7af69d2) SHA1(da050880e186954bcf0e0adf00750dd5a371551b) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6162]: Here too.

File: src/mame/drivers/nmk16.cpp [hunk 10]
```diff
@@ -6125,170 +6111,160 @@ ROM_START( sbsgomo )
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6188]: More ROM regions bunched together.

File: src/mame/drivers/nmk16.cpp [hunk 11]
```diff
@@ -6125,170 +6111,160 @@ ROM_START( sbsgomo )
...
-	ROM_REGION( 0x080000, "oki1", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "91070.3",     0x00000, 0x80000, CRC(ae6875a8) SHA1(bfdb350b3d3fce2bead1ac60875beafe427765ed) ) /* 0x20000 - 0x80000 banked */
-
-	ROM_REGION( 0x080000, "oki2", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "91070.2",     0x00000, 0x80000, CRC(ecfea43e) SHA1(d664dfa6698fec8e602523bdae16068f1ff6547b) ) /* 0x20000 - 0x80000 banked */
-
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91070.3",     0x00000, 0x80000, CRC(ae6875a8) SHA1(bfdb350b3d3fce2bead1ac60875beafe427765ed) ) // 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91070.2",     0x00000, 0x80000, CRC(ecfea43e) SHA1(d664dfa6698fec8e602523bdae16068f1ff6547b) ) // 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6214]: And another.

File: src/mame/drivers/nmk16.cpp [hunk 12]
```diff
@@ -6125,170 +6111,160 @@ ROM_START( sbsgomo )
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6240]: Still one more.

File: src/mame/drivers/nmk16.cpp [hunk 13]
```diff
@@ -6540,139 +6516,137 @@ ROM_START( strahljbl ) // N0 892 PCB, this bootleg uses SEIBU sound system
...
-	ROM_LOAD( "91076-3.45",   0x00000, 0x80000, CRC(b25ed93b) SHA1(d7bc686bbccf982f40420a11158aa8e5dd4207c5) ) /* 1st & 2nd half identical, needs verifying */
-	/* 0x20000 - 0x80000 banked */
+	ROM_LOAD16_WORD_SWAP( "91076-8.57",  0x000000, 0x100000, CRC(7fd0f556) SHA1(d1b4bec0946869d3d7bcb870d9ae3bd17395a231) ) // Sprites
+
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91076-2.46",   0x00000, 0x80000, CRC(3f1e67f2) SHA1(413e78587d8a043a0eb94447313ba1b3c5b35be5) ) // 1st & 2nd half identical, needs verifying
+	// 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91076-3.45",   0x00000, 0x80000, CRC(b25ed93b) SHA1(d7bc686bbccf982f40420a11158aa8e5dd4207c5) ) // 1st & 2nd half identical, needs verifying
+	// 0x20000 - 0x80000 banked
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6540]: These are bunched together now.

File: src/mame/drivers/nmk16.cpp [hunk 14]
```diff
@@ -6540,139 +6516,137 @@ ROM_START( strahljbl ) // N0 892 PCB, this bootleg uses SEIBU sound system
...
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6565]: Another of the same.

File: src/mame/drivers/nmk16.cpp [hunk 15]
```diff
@@ -6540,139 +6516,137 @@ ROM_START( strahljbl ) // N0 892 PCB, this bootleg uses SEIBU sound system
...
 ROM_END
 
-ROM_START( hachamfb ) /* Thunder Dragon conversion - unprotected prototype or bootleg? */
-	ROM_REGION( 0x40000, "maincpu", 0 )     /* 68000 code */
-	ROM_LOAD16_BYTE( "8.bin",  0x00000, 0x20000, CRC(14845b65) SHA1(5cafd07a8a6f5ccbb36de7a90571f8b33ecf273e) ) /* internally reports as 19th Sep. 1991 */
+ROM_START( hachamfb ) // Thunder Dragon conversion - unprotected prototype or bootleg?
+	ROM_REGION( 0x40000, "maincpu", 0 )     // 68000 Code
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:6569]: Code is not a proper noun – there’s definitely no reason to be capitalising it here.

File: src/mame/drivers/nmk16.cpp [hunk 16]
```diff
@@ -7592,93 +7566,115 @@ void afega_state::init_redhawki()
...
+	ROM_LOAD16_BYTE( "7.hex", 0x000000, 0x080000, CRC(5c5b5fa1) SHA1(41946d763f9d72a6322a2f7e3c54a9f6114afe01) )
+
+	ROM_REGION( 0x080000, "bgtile", 0 )   // Layer 0, 16x16x8
+	ROM_LOAD( "4.hex", 0x000000, 0x080000, CRC(03a8d952) SHA1(44252f90e21d6f3841bcdcdac0aba318f94e33b0) )
+
+	ROM_REGION( 0x00100, "fgtile", ROMREGION_ERASEFF )    // Layer 1, 8x8x4
+	// Unused
+
+	ROM_REGION( 0x40000, "oki1", 0 )    // Samples
+	ROM_LOAD( "5.hex", 0x00000, 0x40000, CRC(e911ce33) SHA1(a29c4dea98a22235122303325c63c15fadd3431d) )
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:7609]: Why the `hex` extension?  These aren’t actually Intel hex format are they?  What are the labels or locations on the board?

File: src/mame/mame.lst [hunk 17]
```diff
@@ -32725,7 +32725,8 @@ redhawk                         // (c) 1997 New Vision Ent. license
 redhawkb                        // (c) 1997
 redhawke                        // (c) 1997 Excellent Co., Ltd license
 redhawkg                        // (c) 1997
-redhawki                        // (c) 1997 Haedong Corp. license
+redhawki                        // (c) 1997 Hae Dong Corp. license
+redhawks                        // (c) 1997 Hae Dong Corp. license
```
Comment (cuavas) [inline src/mame/mame.lst:32729]: Do you have a source for this being stylised with the space?  RR rules render names with no space between jaso.

File: src/mame/drivers/nmk16.cpp (hunk 15 above)
Comment (clawgrip) [reply src/mame/drivers/nmk16.cpp:6569]: Fixed, just just here, but also other appearances

File: src/mame/drivers/nmk16.cpp (hunk 16 above)
Comment (clawgrip) [reply src/mame/drivers/nmk16.cpp:7609]: Victor named them as hex, but yes, it's confusing, I've renamed them to bin. The labels were just numbers.

File: src/mame/drivers/nmk16.cpp (hunk 14 above)
Comment (clawgrip) [reply src/mame/drivers/nmk16.cpp:6565]: Fixed them all, thanks

File: src/mame/mame.lst (hunk 17 above)
Comment (clawgrip) [reply src/mame/mame.lst]: Fixed that, and also sorted the new entry alphabetically

**Pull request merged**